blues/application/tasks.py

```diff
@@ -224,24 +224,53 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository().get('url', '')
+    if url or not url.startswith('git@github.com:'):
+        return None
+
+    domain, repo = url.split(':')
+    org, repo = repo.split('/')
+    return org
+
+
+def notify_deploy(role=None, commits=None):
     from .project import project_name, git_repository_path
 
+    msg = '`{deployer}` deployed '
+    if role:
+        msg += '`{project}@{state}:{role}` '
+    else:
+        msg += '`{project}@{state}` '
+
+    org = get_github_org()
+    if org:
+        if commits:
+            msg += '`<https://github.com/{org}/{project}/compare/{from}...{to}|{from}→{to}>` '
+        else:
+            msg += '`<https://github.com/{org}/{project}/commit/{commit}|{commit}>` '
+    else:
+        if commits:
+            msg += '`{from}→{to}` '
+        else:
+            msg += '`{commit}` '
+
+    msg += 'to `{user}@{host}`'
+
+    from_commit, to_commit = commits or (None, None)
     variables = {
         'deployer': git.get_local_commiter(),
         'project': project_name(),
         'state': env.get('state', 'unknown'),
         'role': role,
+        'org': org,
+        'from': from_commit,
+        'to': to_commit,
         'commit': git.get_commit(repository_path=git_repository_path(), short=True),
```

Comment:
We need it only when we don't know `commits`, so an `if` may save some time here.